Geometry/MuonCommonData/python/testExtendedGeometry2026D54XML_cfi.py

```diff
@@ -111,8 +111,8 @@
         'Geometry/CSCGeometryBuilder/data/cscSpecsFilter.xml',
         'Geometry/CSCGeometryBuilder/data/cscSpecs.xml',
         'Geometry/RPCGeometryBuilder/data/2026/v1/RPCSpecs.xml',
-        'Geometry/GEMGeometryBuilder/data/v7/GEMSpecsFilter.xml',
-        'Geometry/GEMGeometryBuilder/data/v7/GEMSpecs.xml',
+        'Geometry/GEMGeometryBuilder/data/v11/GEMSpecsFilter.xml',
```

Comment:
Right now this change is only put in this test file. Will this be propagated to any official detector configurations?

Reply:
Yes, Sundanda will make a workflow for the 16-partition GE2/1 geometry. But before he can do that, I need to update the GEM-CSC trigger, EMTF and GEM validation modules to temporarily reject those 16-partition GE2/1 trigger pads until the code is updated.